editor/editor_vcs_interface.cpp

```diff
@@ -418,8 +418,13 @@ void EditorVCSInterface::create_vcs_metadata_files(VCSMetadata p_vcs_metadata_ty
 		if (f.is_null()) {
 			ERR_FAIL_MSG("Couldn't create .gitattributes in project path.");
 		} else {
-			f->store_line("# Normalize EOL for all files that Git considers text files.");
+			f->store_line("# Properly detect languages on Github")
+			f->store_line("*.gd linguist-language=GDScript")
+			f->store_line("")
```

Comment:
Missing semicolons at the end:

```suggestion
			f->store_line("*.gd linguist-language=GDScript");
			f->store_line("");
```